### ISSUE TITLE
Move the last of the windows DLLs over to this gem

### ISSUE THREAD

**Issue by OJ** [rapid7/meterpreter_bins#11]
We still have the sniffer extension binaries sitting in the `data/meterpreter` folder of the main repo. I'm sure there's a reason for it that might have something to do with it needing the PSSDK to build, but ideally we'd have that baked into the CI build as well.

Is this possible?

**Comment by bcook-r7** [comment]
I think that is the actual intent. I took a look at the last successful CI job on jenkins, and it definitely checked out the pssdk repo adjacent to meterpreter_bins and meterpreter directories.

Something is preventing the auto-detection in make.bat from noticing pssdk and adding the r7_ prefix and building the sniffer dll, but most of the pieces are in place.

**Comment by OJ** [comment]
Good to hear. If there's anything I can do to help facilitate this then gimme a shout! Over time, I'd really like to get _ALL THE METERPRETERZ_ into the gem. I'm also keen to move the other meterpreter payload source (for python, php, etc) into the Meterpreter repo. I'll open an issue over there to start the discussion.

**Comment by todb-r7** [comment]
I agree totally and completely with both of these goals.

**Comment by bcook-r7** [comment]
I will try to get this working correctly today.

**Comment by OJ** [comment]
Sniffer has been moved over. What is the plan with `metcli.exe` and `metsvc-server.exe` ? Should they be moved in as well?

**Comment by bcook-r7** [comment]
Seems logical to me, captain. Except metcli.exe doesn't really seem to work - maybe we should burn it (or fix it).

**Comment by OJ** [comment]
Yeah agreed `metcli.exe` should definitely be removed. I removed the source from the meterpreter repo during the course of updating the source tree to work with VS 2013 aaaaages ago. There's no point in having it in there any more. I'll do a quick PR for that.
